src/main/java/com/example/demo/controller/StringController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.demo.controller;
+
+import com.example.demo.services.StringGeneratorService;
+import com.example.demo.services.StringGeneratorServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class StringController {
+
+    private StringGeneratorService service;
+
+    @Autowired
+    public StringController(StringGeneratorServiceImpl service) {
+        this.service = service;
+    }
+
+    @RequestMapping("/start")
+    public @ResponseBody
```

Comment:
Nie wiem czy używasz jakiegoś auto formatowania, ale to lepiej wygląda jak jest w jednej lini
public @ResponseBody String start() {